std/haxe/macro/Expr.hx

```diff
@@ -602,7 +614,7 @@ typedef TypeParamDecl = {
 	@:optional var params : Array<TypeParamDecl>;
 
 	/**
-		The metadata of the type parameter.
+		The metadata of the type parameter, which is an array of `MetadataEntry`.
```

Comment:
I mean yes, it is, but why mention that explicitly here? We don't do that in any other place...